### PR TITLE
Pie chart

### LINKED ISSUE #346: Pie chart
image

### LINKED ISSUE #373: Expose Legend component
We should expose `Legend` as part of the public API and make it easy to have it live alongside other chart components giving finer control over the placement.

### PR DESCRIPTION
**Pull request by jondlm**
## PR Checklist

- Manually tested across supported browsers
  - [x] Chrome
  - [x] Firefox
  - [x] Safari
  - [x] IE11 (Win 7)
  - [x] Edge (Win 10)
- [x] Unit tests written (`common` at minimum)
- [x] PR has one of the `semver-` labels
- [ ] Two core team engineer approvals
- [x] One core team UX approval

Fixes #346
Fixes #373

### CODE REVIEW

File: src/components/PieChart/PieChart.jsx [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+							/>
+						</g>
+					))}
+				</g>
+			</g>
+		);
+
+		return (
+			<svg
+				{..._.omit(passThroughs, ['onMouseOver', 'onMouseOut'])}
```
Comment (sodiumjoe) [inline src/components/PieChart/PieChart.jsx]: should just `omitProps` i would think

File: src/components/PieChart/PieChart.jsx [hunk 2]
```diff
@@ -0,0 +1,344 @@
...
+				className={cx(className, '&')}
+				width={width}
+				height={height}
+			>
+				{hasToolTips ?
```
Comment (sodiumjoe) [inline src/components/PieChart/PieChart.jsx]: seems slightly cleaner to just move this to `isExpanded`?

```
isExpanded={hasToolTips && isHovering}
```

File: src/components/PieChart/PieChart.jsx (hunk 1 above)
Comment (jondlm) [inline src/components/PieChart/PieChart.jsx]: Totally, I didn't even realize that existed until today.

File: src/components/PieChart/PieChart.jsx (hunk 2 above)
Comment (jondlm) [inline src/components/PieChart/PieChart.jsx]: Well, the other end of this ternary is `: renderPie`.

Comment (jondlm) [inline src/components/PieChart/PieChart.jsx]: Though you might still be right, I might be able to move `renderPie` into the target and do what you said.

Comment (sodiumjoe) [inline src/components/PieChart/PieChart.jsx]: it's already in the target

Comment (sodiumjoe) [inline src/components/PieChart/PieChart.jsx]: and this way you could just inline the `renderPie` definition

**Comment by halseyhemingway** [comment]
@jondlm Can I get a link to an example I can review? Thank you!

**Comment by jondlm** [comment]
@halseyhemingway sure: http://docspot.devnxs.net/projects/lucid/feature-346-pie-chart/#/components/PieChart

**Pull request merged**